### PR TITLE
Pull some wording from the presentation and strawman

### PR DESCRIPTION
**Pull request by twof**
This is the first part of a rewrite before we open up the official RFC PR

### CODE REVIEW

File: rfcs/QueryLevelNullability.md [hunk 1]
```diff
@@ -79,22 +79,23 @@ GraphQL on the client. GraphQL codegen tooling exists for many platforms:
...
+may not be in the best position to decide whether partial failure is acceptable for a given canvas.
 
-While the schema can have nullable fields for valid reasons (such as federation), in some cases the client wants to decide
-if it accepts a `null` value for the result to simplify the client-side logic. In addition, syntax for this concept
-would allow codegen tooling to generate model types that are more ergonomic to work with, since the model
-type's properties would have the desired nullability.
+While the schema can have nullable fields for valid reasons (such as federation), in some cases the client wants 
+to decide if it accepts a `null` value for the result to simplify the client-side logic.
+
+We need another way to make working with GraphQL more ergonomic for client developers.
```
Comment (lizjakubowski) [inline rfcs/QueryLevelNullability.md]: nit: imo we can remove this last sentence -- I don't think it adds anything, but it also shifts the proposal from "client should decide" to "improve developer experience", and I think we should stick to the former throughout the problem statement (even though the latter wrt codegen is an important _use-case_)

File: rfcs/QueryLevelNullability.md [hunk 2]
```diff
@@ -131,16 +132,17 @@ query GetBusinessName($id: String!) {
 
 ### `!`
 
-We have chosen `!` because `!` is already being used in the GraphQL spec to indicate that a field is non-nullable.
-Incidentally the same precedent exists in Swift (`!`) and Kotlin (`!!`) which both use similar syntax to indicate
-"throw an exception if this value is `null`".
+We have chosen `!` because `!` is already being used in the GraphQL spec to indicate that a field in the schema
+is non-nullable. Incidentally the same precedent exists in Swift (`!`) and Kotlin (`!!`) which both use similar
+syntax to indicate "throw an exception if this value is `null`", so the syntax may even feel familiar to 
+non-GraphQL developers.
```
Comment (lizjakubowski) [inline rfcs/QueryLevelNullability.md]: as someone suggested before, let's minimize any mention of Swift/Kotlin in the proposal

File: rfcs/QueryLevelNullability.md [hunk 3]
```diff
@@ -131,16 +132,17 @@ query GetBusinessName($id: String!) {
...
 
 ## Use cases
 
 #### When a field is necessary to the function of the client
```
Comment (lizjakubowski) [inline rfcs/QueryLevelNullability.md]: we should review these use-cases in a separate PR -- as they've evolved over time, some of them now have overlap and should be combined, and others aren't clear/targeted

File: rfcs/QueryLevelNullability.md [hunk 4]
```diff
@@ -79,22 +79,23 @@ GraphQL on the client. GraphQL codegen tooling exists for many platforms:
 
 ## 📜 Problem Statement
 
-The two modern languages used on mobile clients, Swift and Kotlin, are both non-null by default. By contrast, in a
-GraphQL type system, every field is nullable by default. From the [official GraphQL best practice](https://graphql.org/learn/best-practices/#nullability):
+In our experience, client developers have been frustrated that the vast majority of fields are nullable.
```
Comment (lizjakubowski) [inline rfcs/QueryLevelNullability.md:82]: should we rename this file `ClientControlledNullability.md` and re-title it per [Lee's suggestion](https://github.com/graphql/graphql-spec/issues/867#issuecomment-840807186) on the strawman? out of all the feedback we've received, seems like a low-hanging fruit with a big impact 😊 I also wouldn't be opposed to porting over the terminology/definitions he provided

Comment (aprilrd) [reply rfcs/QueryLevelNullability.md:82]: +1 on both changing the name to `ClientControlledNullability` and incorporating the definitions

File: rfcs/QueryLevelNullability.md (hunk 3 above)
Comment (twof) [reply rfcs/QueryLevelNullability.md]: https://github.com/twof/graphql-spec/pull/9

Comment (twof) [reply rfcs/QueryLevelNullability.md]: #10 #11 #12

File: rfcs/ClientControlledNullability.md [hunk 5]
```diff
@@ -79,31 +79,29 @@ GraphQL on the client. GraphQL codegen tooling exists for many platforms:
...
+to decide if it accepts a `null` value for the result to simplify the client-side logic.
 
 ## 🧑‍💻 Proposed Solution
 
 A client specified Non-Null designator.
 
 ## 🎬 Behavior
 
-The proposed query-side Non-Null designator would have identical semantics as the current 
+The proposed query-side Non-Null designator would have identical semantics to the current
```
Comment (lizjakubowski) [inline rfcs/ClientControlledNullability.md]:
```suggestion
The proposed client-controlled Non-Null designator would have identical semantics to the current 
```

File: rfcs/ClientControlledNullability.md [hunk 6]
```diff
@@ -79,31 +79,29 @@ GraphQL on the client. GraphQL codegen tooling exists for many platforms:
...
+The problem with the SDL nonNull (!) is that it eliminates the possibility of partial failure on a given type.
+This forces the SDL author to decide for which fields partial failure is acceptable. A GraphQL schema author 
+may not be in the best position to decide whether partial failure is acceptable for a given canvas.
 
+While the schema can have nullable fields for valid reasons (such as federation), in some cases the client wants 
+to decide if it accepts a `null` value for the result to simplify the client-side logic.
 
 ## 🧑‍💻 Proposed Solution
 
 A client specified Non-Null designator.
```
Comment (lizjakubowski) [inline rfcs/ClientControlledNullability.md]:
```suggestion
A client-controlled Non-Null designator.
```

**Review by lizjakubowski — APPROVED** [review]

**Comment by twof** [comment]
The RFCs moved to the working group repo. Closing in favor of a different PR

**Pull request closed without merging**